src/Sulu/Component/Content/Metadata/Loader/XmlLegacyLoader.php

```diff
@@ -95,6 +95,10 @@ public function load($resource, $type = 'page')
 
         $schemaPath = __DIR__ . static::SCHEME_PATH;
 
+        $cwd = getcwd();
+        // mute errors for non-Windows machines that have chdir disabled by their hosting provider
```

Comment:
Maybe also mention that this is only totally necessary for windows machines, and linux/mac would still work without successfully executing that command. And keep the 120 char limit 😉